tests/test_export.py

```diff
@@ -0,0 +1,39 @@
+from jrnl.exception import JrnlError
+from jrnl.plugins.fancy_exporter import check_provided_linewrap_viability
+
+import pytest
+
+
+@pytest.fixture()
+def datestr():
+
+    yield "2020-10-20 16:59"
+
+
+from textwrap import TextWrapper
```

Comment:
Can you move this import statement to the top of the file with the rest of them?

Reply:
I ended up deleting the import with the unused function `provide_date_wrapper`